R/loggingsummary.R

```diff
@@ -81,31 +80,31 @@ loggingsummary <- function(x
 
   cat('Initial harvestable volume (m3):\n')
   print(unlist(RsltStats[3]))
-  print(set_units(unlist(RsltStats[3])/PlotArea, m3/ha)) # the harvestable volume (m3) with your initial criteria
+  print(set_units(unlist(RsltStats[3])/HarvestableArea, m3/ha)) # the harvestable volume (m3) with your initial criteria
 
```

Comment:
But I thought that you changed that before Vincyane ? WIS it just in this print that it hasn't been corrected or were there still some problem in the actual calculation of the volume objectif ? See discussion from today with Margaux.